### DISCUSSION TITLE
None of the download links on RoutineHub work any longer

### DISCUSSION THREAD

**Discussion by indigodelta** [schlagmichdoch/PairDrop#135] [General]
Clicking on the "Get Shortcut" link or scanning the QR code takes me to a page on iCloud.com that says "Not Found
Unable to find the shortcut.".  None of the older version links work either.  Is this available somewhere else, or am I missing something?

**Comment by schlagmichdoch** [comment]
Seems to be an issue on Apples iCloud service:
https://www.reddit.com/r/shortcuts/comments/15g3ui5/apple_broke_down_the_icloud_server_again/

**Comment by schlagmichdoch** [comment]
Seems to be fixed. @indigodelta does everything work for you again?

**Reply by indigodelta** [reply]
Yep, it works for me now.  Thanks for the help.
